addons/arsenal/functions/fnc_verifyLoadout.sqf

```diff
@@ -19,7 +19,7 @@ private _extendedInfo = createHashMap;
 
 // Check if the provided loadout is a CBA extended loadout
 if (count _loadout == 2) then {
-    _extendedInfo = _loadout select 1;
+    _extendedInfo = +(_loadout select 1); // Copy the hashmap to prevent events from modifiyng the profileNamespace extendedInfo
```

Comment:
If the hashmap is prone to being overwritten in profileNamespace, isn't the array as well?

Reply:
apply returns a copy of the array